phpBB/common.php

```diff
@@ -96,9 +96,9 @@
 require($phpbb_root_path . 'includes/constants.' . $phpEx);
 require($phpbb_root_path . 'includes/utf/utf_tools.' . $phpEx);
 
-if (PHPBB_ENVIRONMENT === 'development')
+if (PHPBB_ENVIRONMENT === 'development' || DEBUG === true)
 {
-	\phpbb\debug\debug::enable();
+	\phpbb\debug\debug::enable(DEBUG_ERROR_LEVEL);
```

Comment:
I don't like passing possibly undefined constants to methods. How about replacing this with something like:
`defined('DEBUG_ERROR_LEVEL') ? DEBUG_ERROR_LEVEL : null`